static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -80,7 +88,7 @@ function GroupReplays({group}: Props) {
           isFetching={isFetching}
           replays={[]}
           sort={undefined}
-          visibleColumns={VISIBLE_COLUMNS}
+          visibleColumns={allMobileProj ? VISIBLE_COLUMNS_MOBILE : VISIBLE_COLUMNS}
```

Comment:
this ternary seems to be used a lot throughout this file, can we make it a constant up top?